src/en/introduction/accessible-image.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Designing accessible images
+layout: layouts/base.njk
+description: With this diagram, you will learn how to make the best choice for designing accessible images.
+---
+        <div class="row">
+            <div class="col-md-10 col-xs-offset-1"><img src="../../../images/en/accessible-image.jpg"
```

Comment:
Image src not right.
https://deploy-preview-144--a11ycanada.netlify.app/en/introduction/accessible-image/